### ISSUE TITLE
Installation Failed because of non-CRAN Dependency

### ISSUE THREAD

**Issue by njlyon0** [cfree14/freeR#1]
Hi! 

I'm trying to install your `freeR` package and having an issue because the `FishLife` dependency is not on CRAN.

When I run the following:
`devtools::install_github("cfree14/freeR")`

I get this error
`ERROR: dependency ‘FishLife’ is not available for package ‘freeR’`

I Googled it and [this Stack Overflow post](https://stackoverflow.com/questions/30493388/create-an-r-package-that-depends-on-another-r-package-located-on-github) says that you need to add `Remotes: James-Thorson/FishLife` to get your package to install correctly on other user's machines.

Thanks!

**Comment by cfree14** [comment]
Yep! Install Jim's FishLife package and then my package should work. Out of curiousity, what're you using it for?

**Comment by njlyon0** [comment]
Gotcha that makes sense, thanks!

My colleague Camila Vargas and I are offering a presentation on quality control in R and we wanted to offer `freeR::complete()` as one method for checking a dataframe for NAs

**Comment by cfree14** [comment]
Oh awesome!
